pkg/generate/app/cmd/newapp.go

```diff
@@ -218,6 +218,11 @@ func (c *AppConfig) SetOpenShiftClient(osclient client.Interface, originNamespac
 		ClientMapper: c.clientMapper,
 		Namespace:    originNamespace,
 	}
+	c.dockerSearcher = app.ImageImportSearcher{
```

Comment:
Sorry you're right. Still not very comforting that we rely on those calls being made in that order.